src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -330,17 +330,17 @@ public C(string a, string b, string c)
     {
         if (string.IsNullOrEmpty(a))
         {
-            throw new ArgumentException(""message"", nameof(a));
+            throw new ArgumentException($""'{nameof(a)}' cannot be null or empty"", nameof(a));
```

Comment:
Why is this feature being changed?

Reply:
it got here by mistake, it's a commit PR #42006